modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -1150,4 +1150,37 @@ definition {
 		ObjectAdmin.assertRequiredErrorPresent();
 	}
 
+	@description = "LPS-171802- Verify if the Condition card is hidden when are using the trigger On Subscription Status Update"
+	@priority = "5"
+	test VerifyTheConditionCardAreHidden {
+		property portal.acceptance = "true";
+
+		task ("Given an object system") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Object",
+				panel = "Control Panel",
+				portlet = "Objects");
```

Comment:
This is the same as _ObjectAdmin.openObjectAdmin();_ except the action is done through the UI instead of URL. You can remove this portion since you already have _ObjectAdmin.openObjectAdmin();_ in your next step.